src/test/java/testsuite/failover/FailoverIntegrationTest.java

```diff
@@ -145,6 +145,8 @@ public void test1_1_failFromWriterToNewWriter_failOnConnectionInvocation()
     final String currentConnectionId = queryInstanceId(testConnection);
     assertTrue(isDBInstanceWriter(currentConnectionId));
     assertNotEquals(currentConnectionId, initialWriterId);
+    System.out.println("currentConnectionId: " + currentConnectionId);
```

Comment:
Not a great practice. Do you need to log it?

Reply:
Should've been removed before commit.